### PR TITLE
fixed two noisy annotation loggers

### LINKED ISSUE #7344: "Annotation will not be calculated" warning is spammed?
### Tool(s) or class(es) involved
HaplotypeCaller of GATK v4.2.0.0

### Description
Hi, I'm using GATK to call variants in Aedes Aegypti samples.
My output is spammed with WARNINGs (see below). I wondered:
1. If this was an issue, i.e. my samples have bad QC/Coverage? Or is it something expected? In particular, there is one sample that seems to generates much more warnings than the others, so that's why I thought it could be a coverage issue?
2. Is there a way to deactivate these warnings, or print them only once?

### Log example (only the first lines, the log is several gigs filled with these warnings)
```
16:16:40.556 WARN  DepthPerSampleHC - Annotation will not be calculated at position 1:18938 and possibly subsequent; genotype for sample TOTO is not called
16:16:40.556 WARN  StrandBiasBySample - Annotation will not be calculated at position 1:18938 and possibly subsequent; genotype for sample TOTO is not called
16:16:40.568 WARN  DepthPerSampleHC - Annotation will not be calculated at position 1:18946 and possibly subsequent; genotype for sample TOTO is not called
16:16:40.568 WARN  StrandBiasBySample - Annotation will not be calculated at position 1:18946 and possibly subsequent; genotype for sample TOTO is not called
16:16:40.726 WARN  DepthPerSampleHC - Annotation will not be calculated at position 1:19264 and possibly subsequent; genotype for sample TOTO is not called
16:16:40.726 WARN  StrandBiasBySample - Annotation will not be calculated at position 1:19264 and possibly subsequent; genotype for sample TOTO is not called
16:16:42.644 WARN  DepthPerSampleHC - Annotation will not be calculated at position 1:20675 and possibly subsequent; genotype for sample TOTO is not called
16:16:42.644 WARN  StrandBiasBySample - Annotation will not be calculated at position 1:20675 and possibly subsequent; genotype for sample TOTO is not called
16:16:42.646 WARN  DepthPerSampleHC - Annotation will not be calculated at position 1:20685 and possibly subsequent; genotype for sample TOTO is not called
16:16:42.647 WARN  StrandBiasBySample - Annotation will not be calculated at position 1:20685 and possibly subsequent; genotype for sample TOTO is not called

```

Thanks in advance!

### PR DESCRIPTION
**Pull request by jamesemery**
Fixes #7344

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/DepthPerSampleHC.java [hunk 1]
```diff
@@ -42,6 +43,7 @@
 @DocumentedFeature(groupName=HelpConstants.DOC_CAT_ANNOTATORS, groupSummary=HelpConstants.DOC_CAT_ANNOTATORS_SUMMARY, summary="Depth of informative coverage for each sample (DP)")
 public final class  DepthPerSampleHC implements GenotypeAnnotation, StandardHCAnnotation, StandardMutectAnnotation {
     private final static Logger logger = LogManager.getLogger(DepthPerSampleHC.class);
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/DepthPerSampleHC.java:45]: Do we still need the normal logger?

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandBiasBySample.java [hunk 2]
```diff
@@ -53,6 +54,7 @@
 @DocumentedFeature(groupName=HelpConstants.DOC_CAT_ANNOTATORS, groupSummary=HelpConstants.DOC_CAT_ANNOTATORS_SUMMARY, summary="Number of forward and reverse reads that support REF and ALT alleles (SB)")
 public final class StrandBiasBySample implements GenotypeAnnotation, StandardMutectAnnotation {
     private final static Logger logger = LogManager.getLogger(StrandBiasBySample.class);
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandBiasBySample.java:56]: same question

**Review by lbergelson — APPROVED** [review]
@jamesemery 1 question, 👍

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/DepthPerSampleHC.java (hunk 1 above)
Comment (droazen) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/DepthPerSampleHC.java:45]: @lbergelson Yes, we do need to keep the regular logger for the other log messages output in these annotation classes -- we only want the "one-shot" logger to apply to the specific warning that was being spammed.

**Review by droazen — APPROVED** [review]
👍 Looks good -- merge when tests pass

**Pull request merged**